### ISSUE TITLE
Some of these icons are using outdated logos.

### ISSUE THREAD

**Issue by maxtron95** [numixproject/numix-core#5135]
I was looking through the icon theme and found some icons that needed to be updated.

- [ ] distributor-logo-backbox
- [ ] disributor-logo-nomadbsd
- [x] distributor-logo-manjaro
- [x] distributor-logo-rhel
- [ ] duolingo
- [ ] paypal
- [ ] teamspeak
- [ ] web-deviantart
- [x] web-google
- [ ] web-pandora
- [ ] web-reddit
- [x] web-unsplash
- [ ] web-yahoo
- [ ] zegrapher

| Icon Name | New Icon |
| --- | :-- |
| `distributor-logo-backbox` | ![](https://www.backbox.org/wp-content/uploads/2018/09/website_backbox_text_black.png) |
| `disributor-logo-nomadbsd` | ![nomadbsd](https://user-images.githubusercontent.com/53346722/80658175-0cca8280-8a4b-11ea-94e7-1b33295b17a2.png) |
| `distributor-logo-manjaro` | ![](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a5/Manjaro_logo_text.png/250px-Manjaro_logo_text.png) |
| `distributor-logo-rhel` | ![](https://upload.wikimedia.org/wikipedia/en/thumb/7/79/Red_Hat_Logo_2019.svg/220px-Red_Hat_Logo_2019.svg.png) |
| `duolingo` | ![image](https://user-images.githubusercontent.com/53346722/77831944-551b1b80-7100-11ea-94f5-64f0057ce23f.png) |
| `paypal` | ![](https://lh3.googleusercontent.com/Y2_nyEd0zJftXnlhQrWoweEvAy4RzbpDah_65JGQDKo9zCcBxHVpajYgXWFZcXdKS_o=s180)
| `teamspeak` | ![](https://lh3.googleusercontent.com/jUgoP67po1y3aQ5iBlBpZkrRrizUNRYAYxonQjrfiHp115V8GlAUdv642Z6qBcOpg9k=s180)
| `web-deviantart` | ![](https://lh3.googleusercontent.com/fGG3iP5lTHE-OSoa17uZrM20qGIp4ew2rSR0xiZaYChp2QZzofSeVfWdP3k61_nzWIY=s180) |
| `web-google` | ![](https://lh3.googleusercontent.com/DKoidc0T3T1KvYC2stChcX9zwmjKj1pgmg3hXzGBDQXM8RG_7JjgiuS0CLOh8DUa7as=s180) |
| `web-pandora` | ![](https://lh3.googleusercontent.com/MoJxQ-5RA73b3Wffj91crqXUb-C5LKRYCADm11KbfykIkmNbicXGDa3g9X62C_buEg=s180) |
|`web-reddit` | ![](https://lh3.googleusercontent.com/8Vw-7MAm558750a4M55fiOlUf7lP2cYnFuqSWynrygIiyEEiQQDa_xxHKYOX83L0UD2T=s180) |
| `web-unsplash` | ![](https://d25hn4jiqx5f7l.cloudfront.net/companies/logos/original/unsplash_1544812782.jpg?1544812782) |
| `web-yahoo` | ![](https://lh3.googleusercontent.com/kEANGYQlzMWTececjc09KCHg0wYg1Ew6CwLELcp8CuMqOAmlKS0ZvdpmXesoCiTJkdky=s180) |
| `zegrapher`| ![](https://raw.githubusercontent.com/AdelKS/ZeGrapher/master/icons/ZeGrapher.png) |

**Comment by Foggalong** [comment]
That new Pandora icon is :fire: but is going to be a pain in the ass to do anything with numix wise

**Comment by really-need-an-api-key** [comment]
Perhaps something like this for web-pandora?
![pandora-square](https://user-images.githubusercontent.com/11636003/108623024-e8f28600-743c-11eb-976e-36d99ce68180.png)

I'm not even sure if the icon wasn't updated again.

**Comment by palob** [comment]
I think it's just a blue **P** now.

**Comment by really-need-an-api-key** [comment]
Their [favicon](https://www.pandora.com/favicon.ico) is still colourful.

![image](https://user-images.githubusercontent.com/11636003/108733396-c345a980-752e-11eb-837b-60cb68c944ee.png)

**Comment by palob** [comment]
OK, maybe straight diagonal lines on the P and a coloured baseplate for a softer contrast?

**Comment by really-need-an-api-key** [comment]
I'm finding it a little difficult to pick a good base colour that works well with those stripe colours.

![pandora-square-2](https://user-images.githubusercontent.com/11636003/108886141-e6448c00-7608-11eb-9d57-1b6ab394c8e2.png)

For the stripes i might forgo the blue and use a blue base, but it stops the logo from feeling "fresh" then, and the similarity of the shape colours make it feel more like PayPal.

![pandora-square-4](https://user-images.githubusercontent.com/11636003/108886530-5ce18980-7609-11eb-8ffd-403a3711e647.png)

I think it would look better if the stripes weren't as regular, hence the waviness in my previous comment. Maybe I could break the regularity by not having every line be parallel.

**Comment by palob** [comment]
I don't think it looks all that bad. Maybe slightly lighter / less saturated colours and a significantly lighter baseplate?

**Comment by palob** [comment]
Given the colours even alter in strip direction maybe there is a way building the P from blocks?

How are the waves aligned w.r.t. the pixel grid?

**Comment by really-need-an-api-key** [comment]
They weren't really, but I adjusted them to be.

![Screenshot_2021-03-09_15-45-28](https://user-images.githubusercontent.com/11636003/110488369-b3ad8f80-80ee-11eb-80c0-9589e7794b0d.png)

**Comment by palob** [comment]
Looks like the way to go!

**Comment by really-need-an-api-key** [comment]
If the colours are okay I'll make a PR.
![pandora-circle](https://user-images.githubusercontent.com/11636003/111035009-81669f80-8418-11eb-9482-365914936890.png)
![panora-square](https://user-images.githubusercontent.com/11636003/111035011-83c8f980-8418-11eb-87d7-9f97611d9ced.png)

**Comment by palob** [comment]
Maybe a bit brighter/less saturated colours on the symbol for a softer contrast and it's good to go.

**Comment by really-need-an-api-key** [comment]
I sat on it for a little while and decided I can do better. What about something like this?
![pandora-circle-alt-4](https://user-images.githubusercontent.com/11636003/111864404-f5e19700-8960-11eb-907c-dcd35fc09914.png)

**Comment by palob** [comment]
It certainly looks cool but on the other hand it's a bit busy and maybe not as close to the original. I'm not sure.
I'd say go ahead with the PR, show the other one as alternative and let have others chime in with their opinions.

**Comment by palob** [comment]
Here you can find some more updated logos from the Google universe:
https://about.google/intl/en/products/#all-products
